src/createServer.js

```diff
@@ -1,3 +1,74 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizeUrl = new URL(req.url, `http://${req.headers.host}`);
+
+    const textToConvert = normalizeUrl.pathname.slice(1);
+
+    const params = Object.fromEntries(normalizeUrl.searchParams.entries());
+
+    const toCase = params.toCase;
+
+    const cases = [
+      'SNAKE',
+      'KEBAB',
+      'CAMEL',
+      'PASCAL',
+      'UPPER',
+    ];
+
+    const ArrOfErrors = [];
```

Comment:
Don't start names from upper case if it is not Component, Class or Enum
```suggestion
    const errorMessages = [];
```

Reply:
fixed, thank you!